### PR TITLE
feat(espower-source): add async-await support

### PR DESCRIPTION
**Pull request by jamestalmage**

### CODE REVIEW

**Comment by jamestalmage** [comment]
@twada 
bump!

File: index.js [hunk 1]
```diff
@@ -73,7 +74,7 @@ function adjustFilepath (filepath, sourceRoot) {
 }
 
 function instrument (originalCode, filepath, options) {
-    var jsAst = esprima.parse(originalCode, {tolerant: true, loc: true});
+    var jsAst = acorn.parse(originalCode, {locations: true, plugins: {asyncawait: true}});
```
Comment (twada) [inline index.js]: Just for curiosity, is acorn work with `asyncawait: true` without `ecmaVersion: 7` ?

**Comment by twada** [comment]
@jamestalmage Thanks!
I'm attending [Nodefest Tokyo 2015](http://nodefest.jp/2015/) now, and going to give a short talk about my [unassert](https://github.com/twada/unassert) series and of course, power-assert.
So my response will be late today, so sorry for that.
I'll talk about you today ;)

File: index.js (hunk 1 above)
Comment (jamestalmage) [inline index.js]: good call!

**Comment by jamestalmage** [comment]
Cool - send me a link to the talk!
(I'm assuming you will be speaking Japanese, so I probably won't understand much of it)

**Comment by twada** [comment]
@jamestalmage So sorry for my late response. I'll release this!

**Comment by twada** [comment]
@jamestalmage And here is the link to my talk at nodefest Tokyo 2015.

- [unassert - encourage reliable programming by writing assertions in production](http://www.slideshare.net/t_wada/unassert)

Written in English, spoken in Japanese :wink:

**Comment by twada** [comment]
Hmm, [tests failing on travis](https://travis-ci.org/power-assert-js/espower-source/builds/90064185). I'll check this later.

**Pull request merged**